### PR TITLE
get third party lib for openjdk test

### PR DESCRIPTION
**Pull request by OscarQQ**
related: https://github.com/AdoptOpenJDK/TKG/issues/63
Signed-off-by: OscarQQ <Yixin.Qian@ibm.com>

### CODE REVIEW

**Comment by OscarQQ** [comment]
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder_TKG/50/  x86-64_linux

**Comment by llxia** [comment]
openj9jtregtimeouthandler.jar is only needed for openj9/ibm. Could we limit on JDK_IMPL? Thanks

File: openjdk/build.xml [hunk 1]
```diff
@@ -24,6 +24,20 @@
...
+		</or>
+		<then>
+			<property name="skipDependency" value=false/>
+		</then>
+		<else>
+			<property name="skipDependency" value=true/>
+		</else>
+	</if>
+	<property name="LIB" value="tohandler_simple"/>
+	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml"/>
```
Comment (llxia) [inline openjdk/build.xml:40]: I was thinking to have an ant target for this. And `dist` depends on `getDependentLibsCheck`

```
<target name="getDependentLibsCheck" >
  	<if>
		<or>
			<equals arg1="${JDK_IMPL}" arg2="ibm"  />
			<equals arg1="${JDK_IMPL}" arg2="openj9" />
		</or>
		<then>
			<property name="LIB" value="tohandler_simple"/>
	                <import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml"/>
                        <antcall target="getDependentLibs"/>
		</then>
	</if>
</target>
```

Comment (OscarQQ) [reply openjdk/build.xml:28]: updated :)

**Comment by llxia** [comment]
Thanks @OscarQQ

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**